### ISSUE TITLE
Prometheus metrics request send notification too

### ISSUE THREAD

**Issue by own3dh2so4** [containrrr/watchtower#838]
**Is your feature request related to a problem? Please describe.**
On each prometheus scraping, a notification is send with the message `Valid token found`.

**Describe the solution you'd like**
In my opinion it should be easy to disable the notifications that are sent when prometheus fetches the metrics

**Describe alternatives you've considered**
For example add a flag to disable send this logs

**Additional context**
In my case I have configured watchtower and send the notifications to Telegram with `containrrr/shoutrrr`, when I started configuring prometheus I see that each 60s (scrapping interval) I recived a Telegram notification with the text `Valid token found`

My configuration is:

```
version: "3.9"

services:
  watchtower:
    image: "containrrr/watchtower:1.1.6"
    container_name: "watchtower"
    restart: unless-stopped
    mem_limit: 50m
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
      - /etc/timezone:/etc/timezone:ro
      - /home/david/.docker/config.json:/config.json:ro
    environment:
      - WATCHTOWER_NOTIFICATIONS=shoutrrr
      - WATCHTOWER_NOTIFICATION_TEMPLATE=${WATCHTOWER_NOTIFICATION_TEMPLATE}
      - WATCHTOWER_NOTIFICATION_URL=${WATCHTOWER_NOTIFICATION_URL}
      - WATCHTOWER_HTTP_API_TOKEN=${WATCHTOWER_HTTP_API_TOKEN}
    command:
      - -i86400
      - --monitor-only
      - --no-startup-message
      - --http-api-metrics
```

**Comment by xpliz** [comment]
I'm experiencing the same issue.

```
watchtower    | time="2021-03-12T00:43:00+01:00" level=info msg="Valid token found."
watchtower    | time="2021-03-12T00:44:00+01:00" level=info msg="Valid token found."
watchtower    | time="2021-03-12T00:45:00+01:00" level=info msg="Valid token found."
watchtower    | time="2021-03-12T00:46:00+01:00" level=info msg="Valid token found."
```

This is getting pushed to telegram with shoutrrr.

**Comment by simonaronsson** [comment]
I'm thinking that `info` is the wrong level for this log. It should have been `debug`. Anyone mind providing a PR?

**Comment by piksel** [comment]
Fixed in #757
